src/aap_eda/services/activation/engine/podman.py

```diff
@@ -379,7 +379,15 @@ def _pull_image(
                     "username": request.credential.username,
                     "password": request.credential.secret,
                 }
-            image = self.client.images.pull(request.image_url, **kwargs)
+
+            image_url_split = request.image_url.split(":")
```

Comment:
Realise now this will be problematic with custom ports and ipv6. Need to re-work it.